CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.11)
```

Comment:
OpenEXR specifies minimum 3.11. since the Half install option uses the OpenEXR CMake to build/install in a non-custom way I bumped it to prevent a failure due to CMake mismatch.

Reply:
The CI build could require 3.11 but OCIO does not. Anyway, that's a minor point.